client/app/core/config.js

```diff
@@ -1,20 +1,33 @@
 /* eslint angular/window-service: "off" */
+import { RootReducer } from '../reducers'
+import {createTransform, persistStore, autoRehydrate} from 'redux-persist'
+import { asyncSessionStorage } from 'redux-persist/storages'
 
 var DEVEL_DOMAINS = [
   'localhost',
   '127.0.0.1',
   '[::1]'
 ]
 
-var isDevel = window._.includes(DEVEL_DOMAINS, window.location.hostname)
-
+const isDevel = window._.includes(DEVEL_DOMAINS, window.location.hostname)
+const hasDevTools = angular.isDefined(window.__REDUX_DEVTOOLS_EXTENSION__)
 /** @ngInject */
-export function configure ($logProvider, $compileProvider, $qProvider) {
+export function configure ($logProvider, $compileProvider, $qProvider, $ngReduxProvider) {
   $logProvider.debugEnabled(isDevel)
   $compileProvider.debugInfoEnabled(isDevel)
 
   // TODO: Remove following line as per: https://docs.angularjs.org/guide/migration#migrate1.5to1.6-ng-services-$compile
   $compileProvider.preAssignBindingsEnabled(true)
-
+  const storeEnhancers = [
+    autoRehydrate()
+  ]
+  if (hasDevTools) {
+    storeEnhancers.push(window.__REDUX_DEVTOOLS_EXTENSION__())
+  }
+  $ngReduxProvider.createStoreWith(RootReducer, [], storeEnhancers)
```

Comment:
you can pass the cached state from the browser instead of an empty array to persist the store across user sessions. (or a full page refresh)